src/ngraph/op/subtract.hpp

```diff
@@ -50,9 +50,43 @@ namespace ngraph
                 virtual void generate_adjoints(autodiff::Adjoints& adjoints,
                                                const NodeVector& deltas) override;
             };
-        }
+        } // namespace v0
+
+        namespace v1
+        {
+            /// \brief Elementwise subtraction operation.
+            class Subtract : public util::BinaryElementwiseArithmetic
```

Comment:
After last nGraph changes (on master) it should be `class NGRAPH_API Subtract`.

Reply:
Ah yes, I've just resolved conflicts and added `class NGRAPH_API Subtract` only in `v0` namespace.
I missed `v1`, thanks:)